src/action_types/admin.ts

```diff
@@ -135,6 +135,10 @@ export default keyMirror({
     PURGE_ELASTICSEARCH_INDEXES_SUCCESS: null,
     PURGE_ELASTICSEARCH_INDEXES_FAILURE: null,
 
+    PURGE_BLEVE_INDEXES_REQUEST: null,
+    PURGE_BLEVE_INDEXES_SUCCESS: null,
+    PURGE_BLEVE_INDEXES_FAILURE: null,
+
```

Comment:
Any reason why you are tracking the request ?

Reply:
No, is not needed, I'll remove it. I was following the pattern of the other purge command (for elasticsearch).

Reply:
Yes, if you  don't need to track is better not to add them, as each action will be dispatch causing multiple unnecessary mapStateToProps to run